BlueprintUICommonControls/Tests/Sources/AccessibilityContainerTests.swift

```diff
@@ -6,7 +6,7 @@ import XCTest
 class AccessibilityContainerTests: XCTestCase {
 
     func test_recursiveAccessibleSubviewsIncludesContainedElements() {
-
+        XCTFail("Test failure")
```

Comment:
I think you'll need to fail a snapshot test to get any test artifacts. This one's just a unit test.

Reply:
Yeah I was noticing it wasn't dirtying that screenshots directly, figured that might be why 👍